shoop/campaigns/models/basket_effects.py

```diff
@@ -18,7 +18,7 @@ class BasketDiscountEffect(PolymorphicShoopModel):
     model = None
     admin_form_class = None
 
-    campaign = models.ForeignKey("BasketCampaign", related_name='effects', verbose_name=_("campaign"))
+    campaign = models.ForeignKey("BasketCampaign", related_name="basket_discount_effects", verbose_name=_("campaign"))
```

Comment:
It's a BasketCampaign so the word `basket` is a bit redundant here. The related name could be just `discount_effects`.